WordPress/Docs/PHP/DisallowShortTernaryStandard.xml

```diff
@@ -0,0 +1,19 @@
+<documentation title="Disallow Short Ternaries">
+    <standard>
+    <![CDATA[
+    The short ternary operator must not be used.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: long ternary.">
+        <![CDATA[
+$height = ! empty( $data['height'] ) <em>?</em> $data['height'] <em>:</em> 0;
+        ]]>
+        </code>
+        <code title="Invalid: short ternary.">
+        <![CDATA[
+$height = $data['height'] <em>? :</em> 0;
```

Comment:
The space between `?` and `:` is intentional?

Reply:
Yes. WPCS has no specific rules about the formatting of short ternaries and considering they will be banned, specifying such rules has low priority.

In the absence of specific rules, WPCS will treat the `?` and `:` the same way as other operators and demands whitespace on both sides.

Reply:
Oh this makes sense, thanks!